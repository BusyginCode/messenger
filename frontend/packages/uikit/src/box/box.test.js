import React from 'react'
import { shallow } from 'enzyme'
import Box from './box'

describe('uikit/Box', () => {
  it('should render correctly', () => {
    expect(
      shallow(
        <Box justify="center" margin="s" padding="m">
          Box
        </Box>
      )
    ).toMatchSnapshot()
  })

  it('should render correctly without inject shorthand', () => {
    expect(
      shallow(
        <Box inject flexGrow="1" flexShrink="1" flexBasis="50%" justify="center" padding="m m l 0" margin="m m l 0">
          <span>Box incorrect</span>
        </Box>
      )
    ).toMatchSnapshot()
  })
})
