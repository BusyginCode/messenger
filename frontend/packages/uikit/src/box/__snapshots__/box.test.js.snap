// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`uikit/Box should render correctly 1`] = `
<div
  className="box justify_center"
  style={
    Object {
      "flex": undefined,
      "flexDirection": undefined,
      "margin": 8,
      "marginBottom": undefined,
      "marginLeft": undefined,
      "marginRight": undefined,
      "marginTop": undefined,
      "order": undefined,
      "padding": 16,
    }
  }
>
  Box
</div>
`;

exports[`uikit/Box should render correctly without inject shorthand 1`] = `
<span
  className="box justify_center"
  style={
    Object {
      "flex": undefined,
      "flexBasis": "50%",
      "flexDirection": undefined,
      "flexGrow": "1",
      "flexShrink": "1",
      "margin": "16px 16px 32px 0 ",
      "marginBottom": undefined,
      "marginLeft": undefined,
      "marginRight": undefined,
      "marginTop": undefined,
      "order": undefined,
      "padding": "16px 16px 32px 0 ",
    }
  }
>
  Box incorrect
</span>
`;
